test_runner/regress/test_compute_http.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
```

Comment:
Do we think this test is even valuable? It'll need some rejigging to include an invalid compute id and an empty compute id

Reply:
If you ask me, I think it's fine to not have a test. But ultimately your choice.